### PR TITLE
[FEATURE] Added Grapqhl Endpoint to Retrieve Last Order Information s…

### PR DESCRIPTION
**Pull request by lewisvoncken**
…he Order Increment can be used in PWA

### Description (*)

When you place an order it should return the Increment Id and not the entity id

### Before changes
image

### After Changes - also implementen in PWA Studio see related PRs
image

### Related PRs
Implementation in PWA Studio
https://github.com/magento-research/pwa-studio/pull/649

This is a port of the Rest API Call but it is only necessary for PWA so it is ported to GraphQl
https://github.com/magento/magento2/pull/19713

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/SalesGraphQl/etc/schema.graphqls [hunk 1]
```diff
@@ -2,7 +2,8 @@
 # See COPYING.txt for license details.
 
 type Query {
-    customerOrders: CustomerOrders @resolver(class: "Magento\\SalesGraphQl\\Model\\Resolver\\Orders") @doc(description: "List of customer orders")
+    customerOrders: CustomerOrders @resolver(class: "Magento\\SalesGraphQl\\Model\\Resolver\\Orders") @doc(description: "List of customer orders"),
+    currentSessionLastOrder: CustomerOrder @resolver(class: "Magento\\SalesGraphQl\\Model\\Resolver\\CurrentSessionLastOrder") @doc(description: "List of customer orders"),
```
Comment (nuzil) [inline app/code/Magento/SalesGraphQl/etc/schema.graphqls]: Please change description of method

**Comment by zetlen** [comment]
@lewisvoncken You can test this locally:

1. Check out this PR into a local instance of Magento
2. Provision that instance with Venia sample data using [this script](https://github.com/magento-research/pwa-studio/blob/release/2.0/packages/venia-concept/deployVeniaSampleData.sh)
3. In your PWA Studio working directory, create a new branch
4. Set the `MAGENTO_BACKEND_URL` value in your `packages/venia-concept/.env` file to your local Magento instance
5. Implement GraphQL queries to this endpoint
6. Run `npm run watch:all` to test

**Comment by lewisvoncken** [comment]
@zetlen I have already done that and updated the implementation in PWA Studio just now ;)

https://github.com/magento-research/pwa-studio/pull/649

**Review by nuzil — CHANGES_REQUESTED** [review]
Can you please also cover this implementation with tests?

File: app/code/Magento/SalesGraphQl/etc/schema.graphqls (hunk 1 above)
Comment (lewisvoncken) [reply app/code/Magento/SalesGraphQl/etc/schema.graphqls]: Thanks for the feedback and I have just done this.

**Comment by zetlen** [comment]
@nuzil You are better qualified than me to review @lewisvoncken's changes. Do you think you have the time in the next week?

**Comment by paliarush** [comment]
Unfortunately we cannot rely on customer checkout session in GraphQL resolvers. Feel free to re-open if alternative solution found.

**Pull request closed without merging**